ui/app/clinical/common/views/patientInfoPrint.html

```diff
@@ -2,17 +2,19 @@
     <table>
         <tr>
             <td>Name: <span class="value">{{patient.name}}</span></td>
-            <td>Age / Sex: <span class="value">{{patient.birthdate | dateToAge:visit.dateStarted}} / {{patient.gender}}</span>
-            </td>
-            <td>Registration No. <span class="value">{{patient.identifier}}</span></td>
+            <td>Age: <span class="value">
+                {{patient.birthdate | dateToAge:visit.dateStarted}}
+                ({{patient.genderText.substring(6, patient.genderText.length - 7)}})
+            </span></td>
+            <td>Registration No: <span class="value">{{patient.identifier}}</span></td>
         </tr>
         <tr>
             <td>Village / Town: <span class="value">{{patient.address.cityVillage}}</span></td>
-            <td>P.O. <span class="value">{{patient.address.address2}}</span></td>
+            <td>P.O.: <span class="value">{{patient.address.address2}}</span></td>
```

Comment:
remove . after P.O. .. as you already introducing :

Reply:
@angshu resolved this issue